src/Maths/math.vector.ts

```diff
@@ -10,6 +10,10 @@ import { PerformanceConfigurator } from '../Engines/performanceConfigurator';
 
 type TransformNode = import('../Meshes/transformNode').TransformNode;
 
+var _ExtractAsInt = (value: number) => {
+    return parseInt(value.toString().replace(/\W/g, ""));

```

Comment:
Don't you want to use `\D` instead of `\W`?

Reply:
nope I want to remove everything which is not a number

Reply:
But then `123.123` will have the same result as `1231.23` or `123123`, no?

Reply:
correct. The goal is not to have a perfect hash as it is impossible but at least to support more cases

Reply:
> nope I want to remove everything which is not a number

That's what `\D` is for, no? `\W` won't remove a-z/A-Z but `\D` will.

`_ExtractAsInt("32a43")` will currently return 32 whereas it will return 3243 if using `\D`.

`_ExtractAsInt("a3243")` will currently return NaN whereas it will return 3243 if using `\D`.

Reply:
wait ;) the vector3 contains float ;)

```
var temp = 0.255454685;
console.log(temp.toString().replace(/\W/g, ""));
```

will return: 255454685

Reply:
This PR is not to support strings in Vector ;)